modules/openapi-generator/src/main/resources/cpp-rest-sdk-client/model-source.mustache

```diff
@@ -28,7 +28,7 @@ web::json::value {{classname}}::toJson() const
     web::json::value val = web::json::value::object();
 
     {{#allowableValues}}{{#enumVars}}
-    if (m_value == e{{classname}}::{{classname}}_{{name}}) val = web::json::value::string(U({{{value}}}));{{/enumVars}}{{/allowableValues}}
+    if (m_value == e{{classname}}::{{classname}}_{{name}}) val = web::json::value::string(U("{{{value}}}"));{{/enumVars}}{{/allowableValues}}
```

Comment:
Would there be any issue if the enum values are numeric (e.g. 3, 4, 5) ?

Reply:
What about overriding `toEnumValue` instead? e.g. https://github.com/OpenAPITools/openapi-generator/blob/5d68bd6a03f0c48e838b4fe3b98b7e30858c0373/modules/openapi-generator/src/main/java/org/openapitools/codegen/languages/AbstractApexCodegen.java#L550-L563

Reply:
@wing328 It seems like numeric values have never been supported with cpprestsdk generator. I am looking for what can be done.
Not sure whether it should be a part of this or separate PR.